irods/client_configuration/__init__.py

```diff
@@ -111,7 +126,38 @@ def save(root = None, string='', file = ''):
         if _file and auto_close_settings:
             _file.close()
 
-def _load_config_line(root, setting, value):
+@contextlib.contextmanager
+def loadlines(entries, common_root = sys.modules[__name__]):
```

Comment:
I would personally appreciate a docstring for this just so it's super clear what function is for.

Reply:
Done, ... docstring provided.